developers/weaviate/configuration/rbac/manage-users.mdx

```diff
@@ -0,0 +1,476 @@
+---
+title: Manage users
+sidebar_label: Manage users
+sidebar_position: 1
+image: og/docs/configuration.jpg
+# tags: ['rbac', 'roles', 'configuration', 'authorization']
+---
+
+import Link from '@docusaurus/Link';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import FilteredTextBlock from '@site/src/components/Documentation/FilteredTextBlock';
+import PyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.permissions.py';
+import UserPyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.users.py';
+import OidcUserPyCode from '!!raw-loader!/_includes/code/python/howto.configure.rbac.oidc.users.py';
+
+:::info Added in `v1.29` and `v1.30`
+Role-based access control (RBAC) is generally available in Weaviate from version `v1.29`.  
+User management is available from version `v1.30`.
+:::
+
+In Weaviate, Role-based access control (RBAC) allows you to define roles and assign permissions to those roles. Users can then be assigned to roles and inherit the permissions associated with those roles.
+
+On this page, you will find examples of how to programmatically **manage users** and their associated roles with Weaviate client libraries.
+
+import ConfigureRbac from '/_includes/configuration/configure-rbac.mdx';
+
+<ConfigureRbac />
+
+## User management <i class="fas fa-user-edit"></i> {#user-management}
+
+### Create a user
+
+This example creates a user called `custom-user`.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+  <FilteredTextBlock
+    text={UserPyCode}
+    startMarker="# START CreateUser"
+    endMarker="# END CreateUser"
+    language="py"
+  />
+  </TabItem>
+  <TabItem value="js" label="JS/TS Client v3">
+
+```ts
+// TS support coming soon
+```
+
+  </TabItem>
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+  <TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+<details>
+  <summary>Example results</summary>
+
+```text
+RXF1dU1VcWM1Q3hvVndYT0F1OTBOTDZLZWx0ME5kbWVJRVdPL25EVW12QT1fMXlDUEhUNjhSMlNtazdHcV92MjAw
+```
+
+</details>
+
+### Delete a user
+
+This example deletes a user called `custom-user`.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python Client v4">
+  <FilteredTextBlock
+    text={UserPyCode}
+    startMarker="# START DeleteUser"
+    endMarker="# END DeleteUser"
+    language="py"
+  />
+  </TabItem>
+  <TabItem value="js" label="JS/TS Client v3">
+
+```ts
+// TS support coming soon
+```
+
+  </TabItem>
+  <TabItem value="go" label="Go">
+
+```go
+// Go support coming soon
+```
+
+  </TabItem>
+  <TabItem value="java" label="Java">
+
+```java
+// Java support coming soon
+```
+
+  </TabItem>
+</Tabs>
+
+### List all users
+
+This example shows how to get a list of all the users in Weaviate.
```

Comment:
e.g. here, we would want to be clear what we mean by `user`. Whether these are just DB users, DB + DB_ENV users, or DB, DB_ENV and OIDC users with roles.